pkgs/top-level/all-packages.nix

```diff
@@ -380,6 +380,10 @@ with pkgs;
 
   coordgenlibs  = callPackage ../development/libraries/coordgenlibs { };
 
+  copilot-cli = callPackage ../tools/admin/copilot-cli {
+    buildGoModule = buildGo118Module; # at least Go 1.18 is required
```

Comment:
That is the default now so we can remove this

Reply:
Removed